lib/index.ts

```diff
@@ -86,9 +94,26 @@ class PWMetrics {
         await sheets.appendResults(results.runs);
       }
     }
+
+    if (resultHasExpectationErrors && this.flags.expectations) {
+      throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
+    }
+
     return results;
   }
 
+  resultHasExpectationErrors(metrics: MetricsResults): boolean {
+    const expectationErrors = metrics.timings.filter((timing: Timing) => {
+      const expectation = this.expectations[timing.id];
+      if (!expectation) {
+        return false;
+      }
+      const expectedErrorLimit = expectation.error;
+      return expectedErrorLimit !== undefined && timing.timing >= expectedErrorLimit;
```

Comment:
Is there any parsing step to `expectation.error`? In the fixture's tests you'd defined it as `'>=3000', but here you've hardcoded the operator and are using the `error` as a number, is it correct?